mlrun/utils/model_monitoring.py

```diff
@@ -156,6 +156,53 @@ def get_connection_string(project: str = None):
         )
 
 
+def get_stream_path(project: str = None):
+    """Get stream path from the project secret. If wasn't set, take it from the system configurations"""
+
+    if is_running_as_api():
```

Comment:
We're looking to enforce better client-server separation in our code now. The entire file here is mixed code that should ideally be avoided. For now it's still in WIP phase, but keep in mind that this entire file will need to go through some refactoring in the near future - let's at least put a TODO here.